cmd/run.go

```diff
@@ -267,20 +267,20 @@ Run sidecar only:
-	RunCmd.Flags().StringVarP(&configFile, "config", "c", standalone.DefaultConfigFilePath(), "Dapr configuration file. Default is $HOME/.dapr/config.yaml or %USERPROFILE%\\.dapr\\config.yaml")
-	RunCmd.Flags().IntVarP(&port, "dapr-http-port", "H", -1, "the HTTP port for Dapr to listen on")
-	RunCmd.Flags().IntVarP(&grpcPort, "dapr-grpc-port", "G", -1, "the gRPC port for Dapr to listen on")
-	RunCmd.Flags().StringVarP(&image, "image", "", "", "the image to build the code in. input is repository/image")
+	RunCmd.Flags().IntVarP(&appPort, "app-port", "p", -1, "The port your application is listening on")
+	RunCmd.Flags().StringVarP(&appID, "app-id", "i", "", "The id for your application, used for service discovery")
+	RunCmd.Flags().StringVarP(&configFile, "config", "c", standalone.DefaultConfigFilePath(), "Dapr configuration file")
+	RunCmd.Flags().IntVarP(&port, "dapr-http-port", "H", -1, "The HTTP port for Dapr to listen on")
+	RunCmd.Flags().IntVarP(&grpcPort, "dapr-grpc-port", "G", -1, "The gRPC port for Dapr to listen on")
+	RunCmd.Flags().StringVarP(&image, "image", "", "", "The image to build the code in (input is repository/image)")
 	RunCmd.Flags().BoolVar(&enableProfiling, "enable-profiling", false, "Enable pprof profiling via an HTTP endpoint")
-	RunCmd.Flags().IntVarP(&profilePort, "profile-port", "", -1, "the port for the profile server to listen on")
-	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "Sets the log verbosity. Valid values are: debug, info, warn, error, fatal, or panic. Default is info")
-	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "controls the concurrency level of the app. Default is unlimited")
-	RunCmd.Flags().StringVarP(&protocol, "app-protocol", "P", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
-	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "d", standalone.DefaultComponentsDirPath(), "Path for components directory. Default is $HOME/.dapr/components or %USERPROFILE%\\.dapr\\components")
-	RunCmd.Flags().String("placement-host-address", "localhost", "the host on which the placement service resides")
-	RunCmd.Flags().BoolVar(&appSSL, "app-ssl", false, "when true, will have Dapr invoke the app over https. default is false")
-
+	RunCmd.Flags().IntVarP(&profilePort, "profile-port", "", -1, "The port for the profile server to listen on")
+	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "The log verbosity. Valid values are: debug, info, warn, error, fatal, or panic")
+	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "The concurrency level of the application, otherwise is unlimited")
+	RunCmd.Flags().StringVarP(&protocol, "app-protocol", "P", "http", "The protocol (gRPC or HTTP) Dapr will use to talk to the application")
+	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "d", standalone.DefaultComponentsDirPath(), "The path for components directory")
```

Comment:
"The protocol (gRPC or HTTP) Dapr uses..." always use present tense, not future tense

Reply:
Fixed.